akka-persistence-typed/src/main/scala/akka/persistence/typed/internal/Running.scala

```diff
@@ -97,6 +110,45 @@ private[akka] object Running {
   import InternalProtocol._
   import Running.RunningState
 
+  import scala.concurrent.duration._
+  import akka.actor.typed.scaladsl.AskPattern._
+
+  private val log = setup.log
```

Comment:
that's not right, it might change MDC stuff so shouldn't be in a `val`. Use different Logger from inside the stream/future